dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -597,5 +597,36 @@ def test_correct_total_sample_size_and_counts_and_mutability(self):
         self.assertEqual(col_one_len, len(data['NAME']))
         self.assertEqual(col_two_len, len(data[' VALUE']))
 
+    def test_save_and_load(self):
+        datapth = "dataprofiler/tests/data/"
```

Comment:
when we unit test for save, we shouldn't actually create a file or load a file. We ideally mock the actual IO.
On save, we validate what would be sent to save is what we would expect. On load, we intercept the open and give it what it would be expected to open.

I can see load being more of an issue, but at least it is a static file. However, when we test right now, a save would actually create a file which would need cleanup, no?

Reply:
That makes sense, currently save would create a file which would require cleanup - I'll look into how I can mock the IO